charts/opentelemetry-operator/templates/crds/crd-opentelemetry.io_opampbridges.yaml

```diff
@@ -1,9 +1,12 @@
----
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
+    cert-manager.io/inject-ca-from: opentelemetry-operator-system/opentelemetry-operator-serving-cert
```

Comment:
do we want to use `{{ .Release.Namespace }}` here?

Reply:
Hm, we should, although this doesn't actually matter if there's no conversion webhook.

Reply:
yeah so let's just remove this for now.

Reply:
Ended up being easier to patch it instead.